react/tic-tac-toe/src/app/screens/Signup/styles.module.scss

```diff
@@ -0,0 +1,60 @@
+@import "../../../scss/colors.scss";
```

Comment:
I think you can make @import 'colors.scss' since it is already in your src/scss folder